Translations/French.csv

```diff
@@ -3584,13 +3584,13 @@ tutorial.reactor_wait_fill_done,Le réacteur a été alimenté et il produit mai
 tutorial.reactor_wait_fill,"En attente de travailleur pour alimenter le réacteur. Il devrait y avoir beaucoup de matières premières autour des zones détruites, mais vous pouvez toujours déconstruire les restes du vaisseau pour en obtenir plus.",,
 tutorial.repair_mass.limit,La réparation de masse est actuellement limitée à 25 tuiles pour la durée du didacticiel. La création d'un grand nombre de tâches pour les travailleurs n'est pas recommandée pour le moment.,,
 tutorial.repair_single,"Il est important de réparer tout dommage dans les zones que nous utiliserons. Vous ne pourrez pas utiliser d'appareils endommagés ou construire des objets sur des sols endommagés. De plus, les structures endommagées perdront de l'oxygène.<br><br>Utilisez l'outil {1} <b>Réparer</b> dans le {0} menu pour commander des réparations.<br><br>Quand {1} <b>Réparer</b> l'outil est actif, les objets endommagés seront <b><color=#cc0000ff>surlignés</color></b>. Si vous ne trouvez aucun objet endommagé à proximité, effectuez un zoom arrière et regardez autour de vous.",,
-tutorial.relocate_bed.1,"J'ai détecté un <nobr><b>{0}</b></nobr> endommagé que nous avons pu réparer et déplacer dans notre zone de survie. <br><br>Réparez l'objet, puis sélectionnez-le et utilisez <b><nobr>{1}</nobr></b>.<br><br>Vous devez être en <b><nobr> Mode de sélection libre</nobr></b> ({2}) pour sélectionner l'objet.",,
+tutorial.relocate_bed.1,"J'ai détecté un <nobr><b>{0}</b></nobr> endommagé que nous pouvons réparer et déplacer dans notre zone de survie. <br><br>Réparez l'objet, puis sélectionnez-le et utilisez <b><nobr>{1}</nobr></b>.<br><br>Vous devez être en mode <b><nobr>Sélection libre</nobr></b> ({2}) pour sélectionner l'objet.",,
 tutorial.relocate_bed.2,Trouvez maintenant un nouvel emplacement pour l'objet et cliquez sur {0} pour commander le déplacement.,,
-tutorial.relocate_bed.3,La relocalisation est maintenant en cours.,,
+tutorial.relocate_bed.3,La déplacement du <nobr><b>Pod de couchage</b></nobr> est maintenant en cours.,,
```

Comment:
1. Please remove the HTML tags
2. You are mentioning a sleeping pod here, it may not necessarily be a sleeping pod, could also be a regular bed. I think this translation should be reverted to the original.

Reply:
Oh ok, I didn’t know :) sorry